themes/openy_themes/openy_rose/templates/paragraph/paragraph--grid-columns.html.twig

```diff
@@ -57,7 +57,9 @@ set classes = [
         <h2>{{ content.field_prgf_clm_headline }}</h2>
       {% endif %}
       <div class="text">{{ content.field_prgf_grid_clm_description }}</div>
-      <div class="more-link">{{ content.field_prgf_clm_link }}</div>
+      {% if paragraph.field_prgf_clm_link.0.uri %}
```

Comment:
It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level